tests/datetime_test.py

```diff
@@ -413,3 +413,29 @@ def test_woy_boundary(self):
             ak.Datetime(ak.date_range("2005-01-01", periods=10, freq="d")).week.to_list(),
             pd.Series(pd.date_range("2005-01-01", periods=10, freq="d")).dt.isocalendar().week.to_list(),
         )
+
+    def test_head_tail_resetting_index(self):
```

Comment:
This should be in `tests/dataframe_test.py`

Reply:
oh my word, i def misread that

Reply:
Should be moved now!